.travis.yml

```diff
@@ -0,0 +1,54 @@
+language : cpp
+
+sudo: required
+
+dist: trusty
+
+services:
+  - mysql
+
+## The following environment variables are specified in 
+#  the repository settings at travis-ci.org
```

Comment:
Any reason why not to specify them here, so any developer can modify them via a PR?